app/src/main/java/com/example/habithelper/fragments/ZipcodeDialogFragment.java

```diff
@@ -74,6 +74,7 @@ private void checkZipcode() {
             @Override
             public void onSuccess(int statusCode, Headers headers, JSON json) {
                 currentUser.put("zipCode", zipString);
+                currentUser.add("badgesEarned", "badge_world_traveler");
```

Comment:
will these override other badges?

Reply:
currentUser.add() appends to a list, whereas currentUser.put() overwrites existing data.